app/tests/e2e/test_verifier.py

```diff
@@ -31,13 +38,18 @@ def create_send_request(connection_id: str, protocol_version: str) -> SendProofR
 async def test_send_proof_request(
     bob_and_alice_connection: BobAliceConnect,
     alice_member_client: AsyncClient,
+    schema_definition: SchemaSendResult,
+    bob_member_client: AsyncClient,
 ):
     # V1
     proof_request_v1 = create_send_request(
-        connection_id=bob_and_alice_connection["alice_connection_id"],
+        connection_id=bob_and_alice_connection["bob_connection_id"],
```

Comment:
It would be good here to add a test with an actor with the role of verifier that has no public did. Or is that already added? I couldn't find any test that tests with the verifier not having a public did.

Reply:
good idea actually. I'll double check but I'm quite sure you're correct and there's no test with the verifier not having a public did

Reply:
this still needs doing (note to self @morrieinmaas )

Reply:
I added a test for this in test_trust_registry_integration (it does a full e2e test of registering tenants, creating connections, creating schema, creating credential definition, issuing credential and finally requesting proof).